scripts/screeps_rl_training_runner.py

```
@@ -1001,6 +1010,105 @@ def execute_simulator_runs(
+        if reason:
+            detail = f"{detail}: {reason}"
+        raise RuntimeError(
+            "pre-scale private-simulator trainability smoke gate did not prove runtime parameter consumption: "
+            f"{detail}"
+        )
+    consumed_tick = int_or_none(consumption.get("consumedTick"))
+    if consumed_tick is None:
+        raise RuntimeError(
+            "pre-scale private-simulator trainability smoke gate did not prove runtime parameter consumption: "
+            "missing numeric consumedTick"
+        )
```

Ruff (0.15.14)

[warning] Avoid specifying long messages outside the exception class

(TRY003)